.eslintrc.cjs

```diff
@@ -28,11 +32,12 @@ module.exports = {
     plugins: ["@typescript-eslint", "react"],
     rules: {
         indent: ["error", 4],
-        "linebreak-style": ["error", "unix"],
+        "linebreak-style": ["error", "linux"],
```

Comment:
Is there a reason why it needs to be `linux`? This is causing the eslint action to fail.

Reply:
No, sorry about that. I was getting errors on my end so I changed it to Windows (which fixed it). When I changed it back, I mistakenly changed it to linux when it should have been unix.